cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -102,6 +102,23 @@ else()
   hunter_fatal_error("TODO: set toolset for boost" WIKI "error.boost.toolset")
 endif()
 
+if("@ANDROID@")
+  set(toolset_version ndk)
+endif()
+
+get_directory_property(defs COMPILE_DEFINITIONS)
+foreach(def ${defs})
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D${def}")
+endforeach()
+
+get_directory_property(include_dirs INCLUDE_DIRECTORIES)
+foreach(include_dir ${include_dirs})
+  set(
+      CMAKE_CXX_FLAGS
+      "${CMAKE_CXX_FLAGS} ${CMAKE_INCLUDE_SYSTEM_FLAG_CXX} ${include_dir}"
```

Comment:
Same here. Just tested "Visual Studio" generator - looks like `CMAKE_INCLUDE_SYSTEM_FLAG_CXX` is empty.